plugins/backend/ttrss/ttrssAPI.vala

```diff
@@ -222,7 +222,7 @@ public class FeedReader.ttrssAPI : GLib.Object {
 
 					for(uint i = 0; i < feed_count; i++)
```

Comment:
Why not just do the for starting from feed_count -1?

Reply:
I first wrote
```
for(uint i = (feed_count - 1); i >= 0; i--)
```
then I realized i was unsigned. Another solution would be to type i as an int instead of an uint. Matter of taste I guess ;-)